### PR TITLE
Updated jaxb-runtime for JAXB 2.3 to 2.3.5

### PR DESCRIPTION
**Pull request by tobiasstadler**

### CODE REVIEW

**Comment by mohanaraosv** [comment]
@highsource Could you please merge this in order to be compatible with Java 17

File: pom.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
 		<jaxb20.version>2.0.5</jaxb20.version>
 		<jaxb21.version>2.1.13</jaxb21.version>
 		<jaxb22.version>2.2.11</jaxb22.version>
-		<jaxb23.version>2.3.1</jaxb23.version>
+		<jaxb23.version>2.3.4</jaxb23.version>
```
Comment (Stephan202) [inline pom.xml:49]: By now version [2.3.5](https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-runtime/2.3.5/) is out.

**Review by Stephan202 — APPROVED** [review]
Although [a workaround exists](https://github.com/highsource/maven-jaxb2-plugin/issues/201#issuecomment-808009784), it would be nice to get this merged.

**Comment by mattrpav** [comment]
v0.15.1 was released with support for v2.3.6

**Pull request closed without merging**